pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -109,9 +109,41 @@ func (p *staticPolicy) Name() string {
 }
 
 func (p *staticPolicy) Start(s state.State) {
-	// Configure the shared pool to include all detected CPU IDs.
-	allCPUs := p.topology.CPUDetails.CPUs()
-	s.SetDefaultCPUSet(allCPUs)
+	if err := p.validateState(s); err != nil {
+		glog.Warningf("[cpumanager] static policy invalid state: %s", err)
+		s.ClearState()
```

Comment:
@flyingcougar @ConnorDoyle It will be better if we `panic` here. Resetting the state doesn't help with providing guarantees with cpu assignment.

Reply:
agree, policy change without draining node should result in panic, will add.

Reply:
👍 good catch @balajismaniam